jsapp/js/components/accountSettings.es6

```diff
@@ -144,6 +152,9 @@ export var AccountSettings = React.createClass({
       background: `#${stringToColor(accountName)}`
     };
 
+    let bHeight = $('textarea#bio').prop('scrollHeight');
```

Comment:
I replaced this with the `react-autosize-textarea` component. Works well, and allows for much cleaner code.